index.html

```diff
@@ -37,7 +37,198 @@ <h3><a class="connect" href="#">LET'S CONNECT</a></h3>
+                        <li class="javascript stack"><button>javascript</button></li>
+                    </ul>
+                </div>
+                <div class="click">
+                    <button class="btn">See Project</button>
+                </div>
+            </div>
+        </div>
+
+        <div class="content contentone">
+            <img class="item item1" src="images/Snapshoot-Portfolio3.png" alt="this is an image">
+            <div class="card">
+                <h2 class="item item2">Multi-Post <br> Stories</h2>
+                <div>
+                    <ul class="row">
+                        <li class="item item3">Canopy</li>
+                        <li><img class="item item4" src="images/Counter.png" alt="this is an image"></li>
+                        <li class="item item5">Back End Dev</li>
+                        <li><img class="item item6" src="images/Counter.png" alt="this is an image"></li>
+                        <li class="item item7">2015</li>
+                    </ul>
+                </div>
+                <p class="item item8">A daily selection of privately<br> personalized reads. no accounts or<br> sign-ups are required.</p>
+                <div class="list">
+                    <ul class="skills">
+                        <li class="html stack"><button>html</button></li>
+                        <li class="css stack"><button>css</button></li>
+                        <li class="javascript stack"><button>javascript</button></li>
+                    </ul>
+                </div>
+                <div class="click">
+                    <button class="btn">See Project</button>
+                </div>
+            </div>
+        </div>
+        <!-- end of section chunks -->
+    
+        </main>
+
+        <section class="about">
+            <div class="part final">
+                <div class="structure">
+                    <h2 id="complete">About <br> Myself</h2>
+                    <p class="last">I'm a software developer and i am open to building solutions in form of software's that will
+                        help build your organisation, company or business and also drive growth and efficiency in your
+                        workspace. feel free to contact me as i will look forward to working with you.
+                    </p>
+            
+                    <h3><a href="#" id="intouch">LET'S CONNECT</a></h3>
+            
+                    <ul class="connect social">
+                        <li><a href="#"><img src="images/twiter.svg" alt="twitter"></a></li>
+                        <li><a href="#"><img src="images/Linkedin.svg" alt="linkedin"></a></li>
+                        <li><a href="#"><img src="images/m.svg" alt="m"></a></li>
+                        <li><a href="#"><img src="images/github.svg" alt="github"></a></li>
+                        <li><a href="#"><img src="images/manito.svg" alt="magnito"></a></li>
+                    </ul>
```

Comment:
-[ Required ] Please consider adding the links to all your social media pages in the `href` attribute, these links might be clicked by recruiters who want to reach out to you so it's important that you provide active links to your social media pages.